atqa-framework/.eslintrc.json

```diff
@@ -0,0 +1,70 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint",
+    "prettier",
+    "import",
```

Comment:
Do we need this plugin for just one rule?

Reply:
yes, it could be extended in the future